src/qt/bitcoingui.cpp

```diff
@@ -846,6 +846,7 @@ void BitcoinGUI::showDebugWindowActivateConsole()
 void BitcoinGUI::showHelpMessageClicked()
 {
     helpMessageDialog->show();
+    helpMessageDialog->raise();//bring to top if obscured
```

Comment:
If the helpMessageDialog has already been presented - this change brings it to the front if the user re-clicks on the help menu item - otherwise it remains obscured by any other windows.